docs/courses/csintro3/orientation/logic.md

```diff
@@ -26,20 +61,83 @@ if (num == 1) {
 }
 ```
 
-### Add more examples of if statements with logic expressions they have seen
+Notice how we have been using curly braces (`{` and `}`) and indentations in our code. This is done for two reasons. The first being that it makes our code more readable. We can look at our code and easily decipher what is part of the if statement and what isn't. The second reason is for functionality. While it is true that our code would run the same if we didn't use indentations, we cannot say the same about curly braces. Curly braces are used to group code together. After the condition (or else), the if statement will accept the first thing that follows as what should run if the condition is true (or false). This can either be one line of code, or lines of code grouped together by curly braces. 
+
+With this in mind, the following is completely valid code
 
 ```typescript
-let word: string = "word";
 let num: number = 1;
+if (num == 1) console.log("num is one!");
+else console.log("num is not one!");
+```
+
+This keeps if statements small and compact, but is usually avoided.
+
+## Student Task 1:
 
-if (word == "word" && num > 0) {
-    if (num < 2) {
-        console.log("I'm here!");
-    } else {
-        console.log("No I'm here!");
-    }
+1. Create a variable that stores a number
+2. Use an `if` statement to perform some comparison on this variable
+3. If the condition is true, print a message to the console
+4. If the condition is false, print a different message to the console
+
+## Booleans
+
+In blocks, we saw that we could store the value of a boolean as a variable so that we could keep track of it. In JavaScript, we do the same.
+
+```block
+let x = true
+```
+
+is equivalent to 
+
+```typescript
+let x: boolean = true;
+```
+
+### Boolean Operators
+
+We saw in blocks a way of performing operations on boolean. This was done with the use of Boolean Operators. 
+
+Here is a table of these operators and their JavaScript equivalents
+
+|Name 	|JavaScript 					|Precedence |
+|:---	|:---:							|:---:		|
+|Not 	|`!bool1`						|Highest	|
+|And	|`bool1 && bool2`				|Medium		|
+|Or 	|`bool1` &vert;&vert; `bool2` 	|Lowest		|
+
+### ~hint
+
+Note: When writing Or statements, you need to use the `|` character. This character is called either a vertical bar or pipe and is usually located below the backspace key.
+
+### ~
+
+The precedence of operators becomes an issue when your have different operators that aren't separated by parenthesis. Like the case of `bool1 && bool2 || !bool3`. 
```

Comment:
"The case of" makes it seem like it's a canonical thing ("Everyone knows this famous equation, by golly"). Maybe "For example, in the case of ..." with below comment, and then change the order (`bool1 && (bool2 || !bool3)`)